docs/UsersGuide.asciidoc

```diff
@@ -364,7 +364,8 @@ Based on comments on the group overview individual builds can be tagged. As
 'build' by themselves do not own any data the job group is used to store this
 information. A tag has a build to link it to a build. It also has a type
 and an optional description. The type can later on be used to distinguish
-tag types.
+tag types. OpenQA does not define further tag types other than _important_ tag.
+User is free to choose any tag he wants for his/her jobs.
```

Comment:
The casing of "openQA" is wrong and I'd also tweak the wording a bit, e.g.:
```suggestion
tag types. Note that openQA does not define further tag types besides the _important_ tag.
However, the user is free to choose any tag type as needed.
```

Reply:
*this change is ready for review*